eZ/Publish/Core/Repository/Values/Content/Content.php

```diff
@@ -70,6 +70,14 @@ public function getVersionInfo()
     /**
      * {@inheritdoc}
      */
+    public function getName($languageCode = null)
+    {
+        return $this->versionInfo->getName($languageCode);
```

Comment:
Convention in the rest of the class is to use versionInfo directly, it's in the class itself, and while not critical it saves an extra method call. If this had been from the outside however I would agreed, getVersionInfo() is more clear and avoids the magic getters you hit when calling from outside.